services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from django.db.models import QuerySet
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    if movie_show_time and movie_id and cinema_hall_id:
+        cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+        movie = Movie.objects.get(id=movie_id)
+        movie_session = MovieSession(
+            show_time=movie_show_time,
+            cinema_hall=cinema_hall,
+            movie=movie
+        )
+        movie_session.save()
+        return movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet | MovieSession:
```

Comment:
`-> Queryset`
Remember:
`.all()`, `.filter()` - returns QuerySet
`.get()`, `.create()`, `.delete()` - return object class, `MovieSession` for example